src/main/java/duke/Event.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    private String at;
+    private LocalDate start_date;
+    private LocalDate end_date;
+    private LocalTime start_time;
+    private LocalTime end_time;
+
+    /**
+     * Constructor for an Event task
```

Comment:
Remember to add a full stop at the end of your method summary 😃

Reply:
Fixed